docs/README.md

```diff
@@ -14,13 +14,14 @@ actions:
 
 A Node.js SDK to interact with [Freshworks public REST APIs](https://developers.freshworks.com/documentation/). This SDK can be used from a Node.js application hosted outside the Freshworks ecosystem, as well as from a serverless Freshworks app.
 
-With the initial release, this SDK supports only the [Freshteam REST API](https://developers.freshteam.com/api/).
+With the initial release, this SDK supports only the [Freshteam REST API](https://developers.freshteam.com/api/) and [Freshservice REST API](https://api.freshservice.com/v2/).
 
 ## API support
 
 | Product        | Status |
 | -------------- | ------ |
 | Freshteam      | ✅     |
+| Freshservice   | ✅     |
```

Comment:
Let's not mark this as a check till the entire product is ready. Or, you can add a note saying "Partially supported"

Reply:
Made changes as requested